src/Fields/Field.php

```diff
@@ -145,7 +144,9 @@ public function output($value = null)
      */
     public function outputInput($value = null, $view = null)
     {
-        $view = $view ? $view : $this->getView();
+        if (! $view) {
+            $view = $this->getView();
+        }
```

Comment:
This is more optimised though. Alternatively it could be changed to `$view = $view ?: $this->getView();` but that wouldn't make the code (very slightly) faster. This is a micro optimisation.